testkit-backend/src/main/java/neo4j/org/testkit/backend/messages/requests/SessionBeginTransaction.java

```diff
@@ -108,12 +124,23 @@ private Transaction transaction( String txId )
         return Transaction.builder().data( Transaction.TransactionBody.builder().id( txId ).build() ).build();
     }
 
-    @Getter
```

Comment:
I would prefer keeping these annotations.

Reply:
Sure I just thought it was cleaner to not give `timeoutPresent` and setter. But I don't mind.